frontend/src/AppContext.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { createContext, Dispatch, useReducer } from 'react';
+import {
+  getLinkUrls,
+  NormalizedPractitioner,
+  normalizeFetchedPractitioners,
+  normalizePractitionerResult
+} from './utils/helpers';
```

Comment:
Again with this API... Annoying to have to introduce all these helper methods. This is one of the pitfalls in developing frontends based on public APIs. They don't always serve your use case well.